java/1.2/modules/ROOT/pages/configure-java-module-acb.adoc

```diff
@@ -0,0 +1,306 @@
+
+=== Configuring the Validate Type Operation
+
+Use this operation to validate that an instance is an `instanceof` a class.
+
+. Click the *Validate type* component on the canvas.
+. In the *General* tab:
+.. Select the *Class* that contains the method to use.
+.. Enter the name of the *Instance* on which the method is invoked.
+.. Select *Accept subtypes* to accept subtypes of a class. 
++
+Unselect this option if the instance has to be of the exact same class. 
+
+== Example
+
+[[creating-java-class]]
+=== Creating a Java Class
+
+Create Java classes containing methods to use in your application flows. The classes you create in your project are not available to other projects in your Anypoint Code Builder workspace.
+
+. In your Anypoint Code Builder project, create a new Java package under `src/main/java`.
+. Create a new Java class called `TaxCalculator.java`, and add the implementation, for example:
++
+[source,java,linenums]
+----
+package com.me;
+
+public class TaxCalculator {
+
+    private Double percentBaseTax = 7.0;
+
+    public Double calculateTax(Double price, Integer percentAdditionalTax) {
+        return price * (this.percentBaseTax + percentAdditionalTax) / 100;
+    }
+
+    public Boolean isTaxFree(Double price) {
+        if (price < 10) {
+            return true;
+        }
+        return false;
+    }
+}
+---- 
++
+In this example code, there are two methods:
+* `calculateTax`
++
+Receives the `price` and `percentAdditionalTax` parameters.
+* `isTaxFree`
++
+Receives the `price` parameter.
+
+=== Configure an HTTP Trigger
+
+. Click the *Listener* component on the canvas UI.
+. (Optional) Click the edit icon to change the name of the *Listener* component. 
+. (Optional) If you created a configuration file for the connector, select it, or add it to the XML:
+
+. In the *General* tab, in the *Path* field: `\calculateTax`.
+. In the *Advanced* tab, set the *Allowed methods* to `POST`.
++
+Your code now looks similar to this:
++
+[source,xml,linenums]
+----
+<mule xmlns="http://www.mulesoft.org/schema/mule/core" xmlns:doc="http://www.mulesoft.org/schema/mule/documentation" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:http="http://www.mulesoft.org/schema/mule/http"
+	xmlns:java="http://www.mulesoft.org/schema/mule/java" xsi:schemaLocation="http://www.mulesoft.org/schema/mule/core http://www.mulesoft.org/schema/mule/core/current/mule.xsd
+	http://www.mulesoft.org/schema/mule/http http://www.mulesoft.org/schema/mule/http/current/mule-http.xsd
+	http://www.mulesoft.org/schema/mule/java http://www.mulesoft.org/schema/mule/java/current/mule-java.xsd">
+  <http:listener-config name="HTTP_listener_config">
+    <http:listener-connection host="0.0.0.0" port="8081" />
+  </http:listener-config>
+  <flow name="name1">
+    <http:listener path="/calculatePath" config-ref="HTTP_listener_config" doc:name="Listener" doc:id="511e4743-a5ed-4e74-8174-ead32ff575ea" allowedMethods="POST" />
```

Comment:
regarding allowedMethods="POST" 
POST?

Reply:
is that the default?

Reply:
@dukesphere There is not a default. This is the method being used for this specific example.

Reply:
If you don't specify a method, then all are allowed.